README.md

```diff
@@ -33,9 +33,11 @@ You can run the unit tests by starting the docker containers:
 
     docker compose up --detach
 
-and then running the tests:
+and then running the tests in the pywb container:
 
-    bundle exec rake
+    docker compose exec pywb bundle exec rake
```

Comment:
Running the tests is easier in the docker container if you don't have Python3, pywb and cdxj-indexer installed.

Reply:
again - not a bad thing to put in the README?

Reply:
Yep, it is actually in the README.